docs/source/reference/websecurity.md

```diff
@@ -1,24 +1,21 @@
 # Security Overview
 
-The **Security Overview** section helps you learn about:
-
-- the design of JupyterHub with respect to web security
-- the semi-trusted user
-- the available mitigations to protect untrusted users from each other
-- the value of periodic security audits
+The **Security Overview** section helps you learn about: the semi-trusted user;
```

Comment:
I think the bulleted list delivers this info better

Reply:
I've tried to make the items on the list reference links to the corresponding headings or even turning it to a table of contents but couldn't get to do it, then thought about this alternative... I agree the bulleted list is simpler and makes it more rapidly comprehensible though.